### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by taehoonlee**
This PR fixes some typos: `the the`, `implementaion`, `seperated`, `formated`, `compatiblity`, `fuction`.

### CODE REVIEW

File: extern/libqhull/io.c [hunk 1]
```diff
@@ -3698,7 +3698,7 @@ coordT *qh_readpoints(int *numpoints, int *dimension, boolT *ismalloc) {
         strncat(qh rbox_command, s, sizeof(qh rbox_command)-1);
     }
     if (!s) {
-      qh_fprintf(qh ferr, 6074, "qhull input error: missing \"begin\" for cdd-formated input\n");
+      qh_fprintf(qh ferr, 6074, "qhull input error: missing \"begin\" for cdd-formatted input\n");
```
Comment (QuLogic) [inline extern/libqhull/io.c]: Better to send these upstream instead.

File: doc/users/github_stats.rst [hunk 2]
```diff
@@ -677,7 +677,7 @@ Pull Requests (1003):
 * :ghpull:`7548`: Clarify to_rgba docstring.
 * :ghpull:`7476`: Sticky margins
 * :ghpull:`7552`: Correctly extend a lognormed colorbar
-* :ghpull:`7499`: Improve the the marker table in markers_api documentation
+* :ghpull:`7499`: Improve the marker table in markers_api documentation
```
Comment (tacaswell) [inline doc/users/github_stats.rst]: I just fixed this in the PR title so the next time we generate this file it will stay fixed :+1:

**Comment by tacaswell** [comment]
@taehoonlee Thanks for the PR!

**Comment by tacaswell** [comment]
@taehoonlee (belated) congratulations :tada: on what I think is your first Matplotlib contribution.  Hopefully we will hear from you again!

**Comment by taehoonlee** [comment]
Thank you @tacaswell!

**Pull request merged**